examples/readme-example/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "example-readme"
+version = "0.1.0"
+edition = "2018"
+publish = false
+
+[dependencies]
+axum = { path = "../.." }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0.68"
+tokio = { version = "1.0", features = ["full"] }
+tracing = "0.1"
+tracing-subscriber = "0.3"
```

The other examples still uses tracing-subscriber 0.2 so I think CI will complain about duplicate dependencies. Anyway I think we should go with 0.2 and update all the examples in one go.

```suggestion
tracing-subscriber = "0.2"
```